### ISSUE TITLE
Placeholder does not disappear

### ISSUE THREAD

**Issue by charvoa** [MessageKit/MessageInputBar#11]
The placeholder does not disappear when I type text into the InputTextView.

**Comment by nathantannar4** [comment]
Yes that will be resolved. I missed something when rewriting `InputBarAccessoryView`
